GoogleChromeLabs/ps-analysis-tool
# Fix: Exemption section disappearing

## Description
Sometimes the exemption reason was being overwritten by requests. This caused UI glitch which lead to exemption reason section in cookie landing page to disappear.

## Relevant Technical Choices
- keep exemption reason as undefined if theres no exemption reason or the exemption reason is `None`

## Testing Instructions
 - Go to `nytimes.com`
 - Open the cookies insight page when the page is loading
 - Search for the Exemption section
 - Wait for the load completely
 - You should now see the section doesnt disappear.

## Additional Information:



## Screenshot/Screencast



---

## Checklist






Fixes #682
